tests/api/rulesets.test.ts

```diff
@@ -14,15 +28,14 @@ describe('getRulesets()', () => {
       { user_id: 0, game_id: '', name: '', rules: {}, ruleset_id: 0 },
     ];
 
-    const mockDb = vi.fn().mockResolvedValueOnce(data);
-
-    // @ts-ignore
-    db.getRulesets = mockDb;
+    mocks.getRulesets.mockResolvedValueOnce(data)
```

Comment:
Here we can use the referenced `mocks` variable from above to change the implementation of `getRulesets`. In this case, we want it to resolve (it is a promise after all) the `data` array just above. This happens _once_. If you were to immediately invoke `getRulesets` again, the implementation would be deleted.
 
 Its a good habit to only mock once to prevent issues where other tests are using the wrong implementation.